_summaries/wang2018dynamic_1.md

```diff
@@ -1,19 +1,95 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: wang2018dynamic
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: zayne-sprague
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of Dynamic Graph CNN
+
+- By utilizing edge features of nearby vertices connecting to another vertex, we can apply CNN operations on them computing local and global feature maps for the point cloud that is invariant to permutation, has local structure, and is easy to implement.
+
+## What are edges in a point cloud?
+
+- Graphs are defined by a local neighbourhood of vertices found by using KNN (which points are closest to the current vertex in question)
+
+- Edges are constructed by drawing directed edges to each vertex in the local graph, including edges pointing to the same vertex.
+    - This should sound familiar -- it's similar to self-attention from transformers 
```

Comment:
I'm having difficulty seeing the parallel to self-attention.  Specifically, in what way the input vectors (the `x_i`'s and `x_j`'s in this paper) are transformed into key/query vectors, and how do they "interact"?  For example, eq. 8 (and the equations before it that describe previous methods) shows that there is no direct dot product between the feature vectors.  So it feels more analogous to deformable convolution rather than self-attention.

Reply:
Ah right, so, it's windowed self-attention (or at least that's what I thought it sounded like).  Mostly because it is selecting a subset of nodes, then applying edges to every node within that neighbourhood (including itself).  That sounded like Self-Attention (mostly because it was assigning an edge weight to itself as well as everything else in the neighbourhood)

Reply:
This was a bit of stretch, if you think it's too loose of a connection I can scrap it.